File: hrubiec_Katheirne_GetMeThere/wear/src/main/java/com/example/katie/hrubiec_katheirne_getmethere/objects/Alarm.java
package com.example.katie.hrubiec_katheirne_getmethere.objects;

import java.io.Serializable;

public class Alarm implements Serializable {

    private final long departureTime;
    private final long arrivalTime;
    private final long durationInTraffic;
    private final String startingLoc;
    private final String endingLoc;
    private final long wakeUpBefore;
    private final long identifier;


    public Alarm(long departureTime, long arrivalTime, long durationInTraffic, String startingLoc, String endingLoc, long wakeUpBefore,long identifier) {
        this.departureTime = departureTime;
        this.arrivalTime = arrivalTime;
        this.durationInTraffic = durationInTraffic;
        this.startingLoc = startingLoc;
        this.endingLoc = endingLoc;
        this.wakeUpBefore = wakeUpBefore;
        this.identifier = identifier;
    }

    @Override
    public String toString() {
        return startingLoc + " - " + endingLoc;
    }


    public long getDepartureTime() {
        return departureTime;
    }

    public long getArrivalTime() {
        return arrivalTime;
    }

    public long getDurationInTraffic() {
        return durationInTraffic;
    }

    public String getStartingLoc() {
        return startingLoc;
    }

    public String getEndingLoc() {
        return endingLoc;
    }

    public long getWakeUpBefore() {
        return wakeUpBefore;
    }

    public long getIdentifier() {
        return identifier;
    }
}

File: hrubiecKatherine_GetMeThere/hrubiecKatherine_GetMeThere/Alarm.swift
//
//  Alarm.swift
//  hrubiecKatherine_GetMeThere
//
//  Created by Katie on 2/20/19.
//  Copyright © 2019 Katie. All rights reserved.
//

import Foundation

class Alarm{
    
    let departureTime : CLong
    let arrivalTime : CLong
    let durationInTraffic : CLong
    let startingLoc : String
    let endingLoc : String
    let wakeUpBefore : CLong
    let identifier : CLong
    let userID : String
    
    init(departureTime : CLong,arrivalTime : CLong,durationInTraffic : CLong,startingLoc : String,
         endingLoc : String,wakeUpBefore : CLong,identifier : CLong,userID : String) {
        self.departureTime = departureTime
        self.arrivalTime = arrivalTime
        self.durationInTraffic = durationInTraffic
        self.startingLoc = startingLoc
        self.endingLoc = endingLoc
        self.wakeUpBefore = wakeUpBefore
        self.identifier = identifier
        self.userID = userID
    }
    
    
}
